resources/public/api/definition.json

```diff
@@ -1,30 +1,5 @@
 {
-  "scopes": [
-    {
-      "key": "read:performance-tests",
-      "name": "Performance Tests",
-      "description": "Performance tests",
-      "confidenceLevel": 50
-    },
-    {
-      "key": "read:performance-tests-100",
-      "name": "Performance Tests 100",
-      "description": "Performance tests confidenceLevel 100",
-      "confidenceLevel": 100
-    },
-    {
-      "key": "read:performance-tests-200",
-      "name": "Performance Tests 200",
-      "description": "Performance tests confidenceLevel 200",
-      "confidenceLevel": 200
-    },
-    {
-      "key": "read:performance-tests-300",
-      "name": "Performance Tests 300",
-      "description": "Performance tests confidenceLevel 300",
-      "confidenceLevel": 300
-    }
-  ],
```

Comment:
These scopes aren't in [api-scope](https://github.com/hmrc/api-scope/blob/main/conf/scopes.json). Do they need to be added there before removing from here?